doc/user/content/overview/arrangements.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Arrangements"
+description: "Understand how Materialize arrangements work."
+menu:
+  main:
+    parent: 'overview'
+    weight: 4
+---
+
+We call the mechanisms that maintain materialized views for Materialize dataflows **arrangements**. Understanding them better can help you make decisions that will reduce memory usage while maintaining performance.
+
+## Materialized views
+
+Before we talk about the arrangements that maintain materialized views, let's review what materialized views are, how they work in traditional databases, and how they work in Materialize.
+
+A view is simply a query saved under a name for convenience; the query is executed each time the view is referenced, without any savings in performance or speed. But some databases also support something more powerful––materialized views, which save the *results* of the query for quicker access.
+
+Traditional databases typically only have limited support for materialized views in two ways: first, the updates to the views generally occur at set intervals, so views are not updated in real time, and second, only a limited subset of SQL syntax is supported. These limitations stem from limited support for incremental updates; most databases are not designed to maintain long-running incremental queries, but instead are optimized for queries that are executed once and then wound down. This means that when the data changes, the materialized view must be recomputed from scratch in all but a few simple cases.
+
+Our raison d’être at Materialize is doing something better than this: Materialize stores materialized views in memory, to make them faster to access, and incrementally updates the views as the data changes, to maintain correctness.
+
+## Dataflows
+
+Materialize can make incremental updates efficiently because it's built on an incremental data-parallel compute engine, [Differential Dataflow](https://timelydataflow.github.io/differential-dataflow/introduction.html), which in turn is built on a distributed processing framework called [Timely Dataflow](https://timelydataflow.github.io/timely-dataflow/).
+
+When you create a materialized view and issue a query, Materialize creates a **dataflow**. A dataflow consists of instructions on how to respond to data input. Once executed, the dataflow computes and stores the result of the SQL query in memory, polls the source for updates, and then incrementally updates the query results when new data arrives.
```

Comment:
>  A dataflow consists of instructions on how to respond to data input.

This isn't wrong, but it might help to specialize it to "how to respond to changes to the input data". The distinction being only that "responding to data input" needs one more leap to "yes, but what about when that changes?" and it is really the latter that the dataflow describes.